X/Xorg_libXmu/build_tarballs.jl

```diff
@@ -34,10 +31,13 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     BuildDependency("Xorg_kbproto_jll"),
+    BuildDependency("Xorg_xproto_jll"),
     BuildDependency("Xorg_util_macros_jll"),
+    Dependency("Xorg_xextproto_jll"),
```

Comment:
Is this a runtime dependency? Xorg proto packages are usually headers-only

Reply:
Also, based on the fact all the git tree shas are the same, that should be an AnyPlatform package: https://github.com/JuliaBinaryWrappers/Xorg_xextproto_jll.jl/blob/fa33c1f7aabc9f2e9ef35c0d142002945c6f48ec/Artifacts.toml. As is on archlinux for example: https://archlinux.org/packages/extra/any/xorgproto/

Reply:
Should I update the `Xorg_xextproto` package to an `AnyPlatform` package?

Reply:
I think that'd be better, but not a requirement for this, it shouldn't change anything in practice.

Reply:
Done.